drivers/regulator/regulator_shell.c

```diff
@@ -364,6 +364,33 @@ static int cmd_errors(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_dvsset(const struct shell *sh, size_t argc, char **argv)
+{
+	const struct device *dev;
+	int ret;
```

Comment:
```suggestion
	int ret = 0;
```
shell_str* functions do not set error code on success, you need to initialise it.